src/styles/components/_dropdown.scss

```diff
@@ -78,8 +77,8 @@
 
       &:focus,
       &.focus {
```

Comment:
Ik zie dat we zelf op zich de outline niet wegstijlen, maar... bootstrap doet dat wel!
```
// Prevent the focus on the dropdown toggle when closing dropdowns
.dropdown-toggle:focus {
  outline: 0;
}
```

Reply:
En nu?

Reply:
nu niks. dit volgt ooit. maar niet nu.